src/App.jsx

```diff
@@ -1,7 +1,17 @@
 import data from "./data.json";
+import { Album } from "./components/Album.jsx";
+import { Header } from "./components/Header.jsx";
 
-console.log(data);
 
+console.log("This is the JSON-data",data);
```

Comment:
The following comment doesn't have anything to do with the project, but I just wanted to say that I LOVE how you write what kind of data it is in the console, what a great tip!!